code/drasil-example/Drasil/SSP/DataDefs.hs

```diff
@@ -183,7 +194,7 @@ ratioVarEqn = completeCase [case1, case2]
         case2 = (sin (sy QM.pi_ * ((inxi slipDist - idx (sy slipDist) 0) /
                 (indxn slipDist - idx (sy slipDist) 0))), UnaryOp Not (sy constF))
 
---DD12
+--DD13
```

Comment:
I'm not sure there is much value to comments with hard-coded numbers.  I can see you are making an effort to keep them in sync, but it is a challenge for maintenance.  The numbers are something that are automatically generated in the tex file.  We should either delete these comments, or change them to something meaningful, like the name of the DD that is being defined.

Reply:
Ok then. I've updated the comments by removing the numbers and adding in the DD name and label like so. 
```haskell
--DD ratioVariation: interslice normal to shear force ratio variation function

ratioVariation :: DataDefinition
ratioVariation = dd ratioVarQD [makeCite fredlund1977] Nothing 
  "ratioVariation" []

ratioVarQD :: QDefinition
ratioVarQD = mkQuantDef scalFunc ratioVarEqn

ratioVarEqn :: Expr
ratioVarEqn = completeCase [case1, case2]
  where case1 = (1, sy constF)

        case2 = (sin (sy QM.pi_ * ((inxi slipDist - idx (sy slipDist) 0) /
                (indxn slipDist - idx (sy slipDist) 0))), UnaryOp Not (sy constF))
```
Does that work @smiths?